File: A's/282A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int t; cin >> t;
    int res = 0;

    while (t--) {
        string w;
        cin >> w;

        if (w.front() == *"-" || w.back() == *"-") { res--; }
        else { res++; }
    }
    cout << res;

    return 0;
}

File: A's/py/282A.py
def main() -> None:
    t: int = int(input())
    res: int = 0

    while t:  # > 0
        w: str = str(input())

        if w[0] == "-" or w[len(w) - 1] == "-": res -= 1
        else: res += 1

        t -= 1
    
    print(res)

if __name__ == "__main__":
    main()